src/lib/airtable/patches/patch-01.patch

```diff
@@ -0,0 +1,266 @@
+diff --git a/src/lib/airtable/request.js b/src/lib/airtable/request.js
```

Comment:
I don't know much about patches overall so feel free to ignore this but would it be helpful to rename this to something more descriptive, esp if there could potentially be multiple patches for different purposes?

Reply:
This naming convention was chosen so that patches are run in the correct order (in the case of dependent patches). Hopefully this shouldn't be too much of an issue since there shouldn't be many patches here. Maybe it'd be good to document exactly what each patch does somewhere in the doc. I'll leave that as a future todo, thanks!